config/routes.rb

```diff
@@ -18,7 +18,7 @@
   resources :summarys, only: [] do
     collection do
       get :monthly, to: "summarys/monthly#index"
-      get :shop, to: "summarys/shop#index"
+      get 'shops/index/(:page)', to: 'shops#index', as: 'shops_index'
```

Comment:
**[standard]** <Style/StringLiterals> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
```suggestion
      get 'shops/index/(:page)', to: 'shops#index', as: "shops_index"
```